src/Money.php

```diff
@@ -515,6 +515,66 @@ public function jsonSerialize()
         ];
     }
 
+    /**
+     * @param Money   $first
+     * @param Money[] $collection
+     *
+     * @return Money
+     */
+    public static function minimum(self $first, self ...$collection)
+    {
+        $min = $first;
+
+        foreach ($collection as $money) {
+            if ($money->lessThan($min)) {
+                $min = $money;
+            }
+        }
+
+        return $min;
+    }
+
+    /**
+     * @param Money   $first
+     * @param Money[] $collection
+     *
+     * @return Money
+     */
+    public static function maximum(self $first, self ...$collection)
+    {
+        $max = $first;
+
+        foreach ($collection as $money) {
+            if ($money->greaterThan($max)) {
+                $max = $money;
+            }
+        }
+
+        return $max;
+    }
+
+    /**
+     * @param Money   $first
+     * @param Money[] $collection
+     *
+     * @return Money
+     */
+    public static function sum(self $first, self ...$collection)
+    {
+        return $first->add(...$collection);
+    }
+
+    /**
+     * @param Money   $first
+     * @param Money[] $collection
```

Comment:
This isn't correct. The second argument is not an array. I believe I saw ...Money notation somewhere, but I'm not sure.